src/ocm/hooks/useClusterSupportedPlatforms.ts

```diff
@@ -17,12 +17,13 @@ export default function useClusterSupportedPlatforms(clusterId: string) {
   });
 
   const isLoading = !error && !data;
+  const platformsSupported = ['vpshere', 'nutanix'];
```

Comment:
As mentioned before, we should expose this constant in a reusable place, and it should be used in `isClusterPlatformTypeVM` to identify when there is no active platform integration.